src/nidmm/metadata/functions.py

```diff
@@ -763,7 +762,7 @@
             {
                 'direction': 'in',
                 'enum': None,
-                'name': 'action',
+                'name': 'Action',
```

Comment:
Is this how NI-DMM's CVI FP file has it? Why did this change?
This probably should be filed as an NI internal bug.

Reply:
Yes, this matches the FP file. All parameter names use title case.